common-tasks/device-handling.md

```diff
@@ -114,6 +114,6 @@ If no specific `sourceVM:deviceID` combination is given, *all devices of that DE
 [PCI]:/doc/pci-devices-in-qubes-R4.0/
 
 [security considerations]: /doc/device-considerations/
-[device manager icon]: https://raw.githubusercontent.com/hrdwrrsk/adwaita-xfce-icon-theme/master/Adwaita-Xfce/22x22/devices/media-removable.png <!--TODO: find actual icon used in qubes!-->
+[device manager icon]: https://raw.githubusercontent.com/GammaSQ/qubes-attachment/doc-device-rework/wiki/Devices/media-removable.png
```

Comment:
You should use website's link, not source one. The whole qubes-attachment repository is available at `/attachment/` path, so it will be `/attachment/wiki/Devices/media-removable.png`.

Reply:
This existed solely to be independent of the PR to `qubes-attachment`, fixed now.

However, do you want me to replace links to other media with links to `qubes-attachment` as well? e.g. in `pci-devices.md`:

    [appmenu]: https://raw.githubusercontent.com/QubesOS/qubes-artwork/master/icons/32x32/apps/qubes-appmenu-select.png
    [domain manager icon]: https://raw.githubusercontent.com/QubesOS/qubes-artwork/master/icons/32x32/apps/qubes-logo-icon.png

Reply:
Yes, that would be better.